modules/gltf_blend/doc_classes/EditorSceneFormatImporterBlend.xml

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<class name="EditorSceneFormatImporterBlend" inherits="EditorSceneFormatImporter" version="4.0">
+	<brief_description>
+		EditorSceneFormatImporterBlend imports ".blend" files.
+	</brief_description>
+	<description>
+		EditorSceneFormatImporterBlend imports ".blend" files through the glTF2 3d import pipeline.
+
+		The "filesystem/blender/blender_path" editor setting sets the location of the blender binary.
+
+		The "filesystem/blender/enabled" toogle can be found in the project setting.
+
+		Blender export using glTF2 "use keep separate" mode requires Blender 3.0 and above to reference external textures.
```

Comment:
No empty lines in the XML.
```suggestion
		Imports Blender scenes in the [code].blend[/code] file format through the glTF 2.0 3D import pipeline. This importer requires Blender to be installed by the user, so that it can be used to export the scene as glTF 2.0.
		The location of the Blender binary is set via [member EditorSettings.filesystem/blender/blender_path].
		This importer is only used if [member ProjectSettings.filesystem/blender/enabled] is enabled, otherwise [code].blend[/code] present in the project folder are not imported.
		Blender export using glTF 2.0 "use keep separate" mode requires Blender 3.0 and above to reference external textures.
```
Please check the exact words and case style of "use keep separate" in the Blender 3.0 glTF export settings to use the same here.